enterprise/backend/test/metabase_enterprise/sandbox/query_processor/middleware/row_level_restrictions_test.clj

```diff
@@ -330,7 +335,8 @@
   not normally have FK tests ran for it."
   []
   (cond-> (mt/normal-drivers-with-feature :nested-queries :foreign-keys)
-    (@tx.env/test-drivers :bigquery) (conj :bigquery)))
+    (@tx.env/test-drivers :bigquery) (conj :bigquery)
+    :always                          (disj :presto-jdbc)))
```

Comment:
`:always` is a bit weird here, everywhere else in the codebase we do this sort of thing we use `true`. I don't think there's an agreed-upon idiomatic key to use here like there is for `cond`, but we can at least stick to a common project-wide idiom

Reply:
I think I originally got it from [here](https://clojuredocs.org/clojure.core/cond-%3E#example-570c2e57e4b0fc95a97eab4a), but sure, I'll change to `true`.